pkg/blob/controllerserver.go

```diff
@@ -667,6 +681,94 @@ func (d *Driver) DeleteBlobContainer(ctx context.Context, subsID, resourceGroupN
 	})
 }
 
+// CopyBlobContainer copies a blob container in the same storage account
+func (d *Driver) CopyBlobContainer(ctx context.Context, req *csi.CreateVolumeRequest, accountKey, dstContainerName, storageEndpointSuffix string, secrets map[string]string) error {
+	resourceGroupName, accountName, srcContainerName, _, subsID, err := GetContainerInfo(req.GetVolumeContentSource().GetVolume().GetVolumeId())
```

Comment:
if only volumeID is required in this func, just pass volumeID into this function is better, and also do nil pointer check